flopy/plot/plotutil.py

```diff
@@ -2591,3 +2593,350 @@ def parse_modpath_selection_options(
         tep = ep.copy()
 
     return tep, istart, xp, yp
+
+
+# define minimum expected fields
+_mp7_pathline_fields = ["x", "y", "z", "time", "k", "particleid"]
+_prt_pathline_fields = [
+    "x",
+    "y",
+    "z",
+    "t",
+    "trelease",
+    "imdl",
+    "iprp",
+    "irpt",
+    "ilay",
```

Comment:
Should these conversion routines be moved into mp7particledata instead of sitting in plotutil? Are there any use cases other than plotting, or is plotting the main use of these methods? Do we have export routines that use these too?

Reply:
moving to mp7particledata produces a circular import &mdash; discussed offline, if these functions are more widely used in future they could be moved